src/App.jsx

```diff
@@ -13,28 +13,38 @@ import AppLayout from '@/layouts/AppLayout';
 import { RegisterStaff } from '@/pages/app/RegisterStaff';
 import { AuthProvider, ProtectedRoute } from '@/components/auth-provider';
 import { ClinicCreation } from '@/pages/ClinicCreation';
-import { ClinicaEdicion } from '@/pages/EditClinic';
-import { ClinicaCompletada } from '@/pages/SuccessPage';
+import  {ClinicaEdicion} from '@/pages/EditClinic';
+import {ClinicaCompletada} from '@/pages/SuccessPage';
```

Comment:
this won't import correctly

```suggestion
import { ClinicaEdicion } from '@/pages/app/EditClinic';
import { ClinicaCompletada } from '@/pages/app/SuccessPage';
```